packages/fxa-react/components/Checkbox/checkmark.svg

```diff
@@ -0,0 +1 @@
+<svg height="12" viewBox="0 0 12 12" width="12" xmlns="http://www.w3.org/2000/svg"><path d="m3.98771 11.9923c-.26519-.0001-.51951-.1055-.707-.293l-2.999998-3.00003c-.3789722-.39238-.3735524-1.01608.012181-1.40182.385734-.38573 1.009437-.39115 1.401817-.01218l2.157 2.157 6.31599-9.022997c.3186-.4465571.9375-.553074 1.3871-.23873s.562.932187.2519 1.384727l-6.99999 10.00003c-.16865.2438-.43677.4002-.732.427-.02898.0014-.05802.0014-.087 0z" /></svg>
```

Comment:
I don't have a problem with this, but also, why?

Reply:
It's probably not very maintainable since SVGs are usually are given to us with a width/height. It's so that if they're to be reused we can pass in those values (and be explicit about it in the component to know what size it's going to render at) but I think this could also serve as a default if passing them later just overwrites them.

Reply:
AFAICT it just overwrites it. But I'm fine either way. Will update.